swift/apple/FirezoneKit/Sources/FirezoneKit/Managers/TunnelManager.swift

```diff
@@ -273,7 +291,76 @@ public class TunnelManager {
         callback(self.resourcesListCache)
       }
     } catch {
-      Log.app.error("Error: sendProviderMessage: \(error)")
+      Log.app.error("Error: \(#function): \(error)")
+    }
+  }
+
+  func getLogFolderURL() async -> URL? {
```

Comment:
These will be refactored in a subsequent PR. I think we need to move these to a dedicated IPC struct or entity and simplify the call structure.